test/functional/tests/cache_ops/test_multistream_seq_cutoff.py

```diff
@@ -0,0 +1,250 @@
+        Udev.disable()
+        core = cache.add_core(core_disk)
+
+    with TestRun.step(f"Set seq-cutoff policy to always and threshold to 512KiB."):
+        core.set_seq_cutoff_policy(SeqCutOffPolicy.always)
+        core.set_seq_cutoff_threshold(Size(512, Unit.KibiByte))
+
+    with TestRun.step("Reset core statistics counters."):
+        core.reset_counters()
+
+    with TestRun.step("Run I/O"):
+        stream_size = core_disk.size / 256
+        sequential_streams = streams_seq_rand[0]
+        random_streams = streams_seq_rand[1]
+        fio = (Fio().create_command()
+               .io_engine(IoEngine.libaio)
+               .block_size(Size(1, Unit.Blocks4096))
+               .direct()
+               .offset_increment(stream_size))
+
+        for i in range(0, sequential_streams + random_streams):
+            fio_job = fio.add_job(job_name=f"stream_{i}")
+            fio_job.size(stream_size)
+            fio_job.target(core.system_path)
+            if i < sequential_streams:
+                fio_job.read_write(ReadWrite.write)
+            else:
+                fio_job.read_write(ReadWrite.randwrite)
+
+        pid = fio.run_in_background()
+        while TestRun.executor.check_if_process_exists(pid):
+            sleep(5)
+            TestRun.LOGGER.info(f"{core.get_statistics()}")
+
+    with TestRun.step("Stop cache."):
+        cache.stop()
+
+
+@pytest.mark.parametrizex("streams_seq_rand", [(64, 64), (64, 192)])
+@pytest.mark.parametrizex("filesystem", Filesystem)
+@pytest.mark.parametrizex("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_multistream_seq_cutoff_stress_fs(streams_seq_rand, filesystem, cache_mode):
+    """
+    title: Stress test for multistream sequential cutoff on the device with a filesystem
+    description: |
+        Testing the stability of a system when there are multiple sequential and random I/O streams
+        running against the CAS device with a filesystem when the sequential cutoff policy is set to
+        always and the sequential cutoff threshold is set to a value which is able to be gained by
+        sequential I/O streams.
+    pass_criteria:
+        - No system crash
+    """
+    mount_point = "/mnt"
+    with TestRun.step("Prepare devices. Create filesystem on core device."):
+        cache_disk = TestRun.disks['cache']
+        core_disk = TestRun.disks['core']
+        core_disk.create_filesystem(filesystem)
+
+    with TestRun.step("Start cache and add core."):
+        cache = casadm.start_cache(cache_disk, cache_mode, force=True)
+        Udev.disable()
+        core = cache.add_core(core_disk)
+
+    with TestRun.step("Mount core."):
+        core.mount(mount_point)
+
+    with TestRun.step(f"Set seq-cutoff policy to always and threshold to 512KiB."):
```

Comment:
Wrong value in step description (or below in `set_seq_cutoff_threshold` method).